Assignment-1-July-18/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>JSON Identifier</title>
+    <h1>Welcome to JSON Identifier</h1>
+    <link rel="stylesheet" href="style.css" />
+    <script>
+      function displayPostResponse(response) {
+        response
+          .then((response) => {
+            return response.json();
+          })
+          .then((jsonResponse) => {
+            const uniqueId = jsonResponse.data.id;
+            document.getElementById(
+              "responseBox"
+            ).innerHTML = `Successfully added in DB with ID ${uniqueId}`;
+          })
+          .catch(() => {
+            document.getElementById("responseBox").innerHTML =
+              "Some Error Encountered";
+          });
+      }
+
+      function sendData() {
+        //Fetch API - POST usage
+        const input = document.getElementById("inputResponse").value;
+
+        if (
+          (() => {
```

Comment:
why we add the logic here, I guess it was correct previously in another function. this violates the rule of SRP.